annual-chapter-report.form

```diff
@@ -0,0 +1,20 @@
+# PyLadies Annual Chapter Activity Report
+
+Please share with us summary of your group's activities in the past year, and celebrate!
+
+Must be logged in using your pyladies.com email address to fill in this form.
```

Comment:
*nit*:

`To use this form, you must be logged in using your pyladies.com email address.`